File: app/src/main/java/br/com/dalcim/picpay/data/Transaction.java
package br.com.dalcim.picpay.data;

import com.google.gson.annotations.SerializedName;

/**
 * @author Wiliam
 * @since 25/08/2017
 */
public class Transaction {
    private long id;
    private long timestamp;
    private double value;
    @SerializedName("destination_user")
    private User destinationUser;
    private boolean success;
    private String status;

    public Transaction() {
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(long timestamp) {
        this.timestamp = timestamp;
    }

    public double getValue() {
        return value;
    }

    public void setValue(double value) {
        this.value = value;
    }

    public User getDestinationUser() {
        return destinationUser;
    }

    public void setDestinationUser(User destinationUser) {
        this.destinationUser = destinationUser;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/br/com/vdsantana/picpaytest/transaction/Transaction.kt
package br.com.vdsantana.picpaytest.transaction

import br.com.vdsantana.picpaytest.users.User
import com.google.gson.annotations.SerializedName

/**
 * Created by vd_sa on 31/03/2018.
 */
data class Transaction(
        @SerializedName("destination_user")
        val destinationUser: User,
        val id: Number,
        val status: String,
        val success: Boolean,
        val timestamp: Number,
        val value: Number
)